src/Workspaces/CSharp/Portable/Classification/Classifiers/NameSyntaxClassifier.cs

```diff
@@ -137,8 +137,7 @@ private static bool IsNamespaceName(NameSyntax name)
                 if (symbol.Kind == SymbolKind.Alias)
                 {
                     symbol = (symbol as IAliasSymbol).Target;
-                }
-                else if (symbol.IsConstructor())
+                } else if (symbol.IsConstructor() && name.Parent.Kind() == SyntaxKind.Attribute)
```

Comment:
suggestion: you can use the [IsParentKind()](http://source.roslyn.io/#Microsoft.CodeAnalysis.CSharp.Workspaces/Extensions/SyntaxNodeExtensions.cs,7616fedcf6c8707d) extension here